src/components/Navigation/Navigation.styles.tsx

```diff
@@ -41,16 +41,14 @@ export const StyledFontAwesomeIcon = styled(FontAwesomeIcon)`
     color: ${themes.color.secondary};
 `;
 
-export const SingleNavItem = styled(Link)<SingleNavItemProps>`
-    display: ${props => (props.isShow ? 'block' : 'none')};
+export const SingleNavItem = styled.div<SingleNavItemProps>`
     padding: 0 5px;
```

Comment:
Instead of hardcoding padding, maybe you should add spacing to theming and use it instead of px? And consider using rem/em instead of px.